src/scrabble.js

```diff
@@ -1,5 +1,101 @@
-function scrabble() {
+const pointValuesForLetters = [
+  [1, 'A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'],
+  [2, 'D', 'G'],
+  [3, 'B', 'C', 'M', 'P'],
+  [4, 'F', 'H', 'V', 'W', 'Y'],
+  [5, 'K'],
+  [8, 'J', 'X'],
+  [10, 'Q', 'Z']
+]
+let score = 0
 
+// // function takes any word and returns scrabble score
+function scrabble(word) {
+  // word to upper case
+  const givenWord = caseSensitive(word)
+  // UC word split to array of letters
+  const givenLetters = splitLetters(givenWord)
+  // for each of the given letters, loop ref. array
+  letterLookingFor(givenLetters)
 }
 
+// function turns all given letters to capitals
+const caseSensitive = (aString) => {
+  if (typeof aString === 'string') {
+    return aString.toUpperCase()
+  } else {
+    return ''
+  }
+}
+
+// split UC word into Array of Letters
+const splitLetters = (givenWord) => {
+  const givenLetters = givenWord.split('')
+  console.log('givenLetters =', givenLetters)
+  return givenLetters
+}
+
+// for each element of givenLetters, loop through reference array
+const letterLookingFor = (letterToCompare) => {
+  const questionedLetter5 = letterToCompare
+  letterToCompare.forEach(() => {
+    searchMainArray(pointValuesForLetters, questionedLetter5)
+  })
+}
+
+// look through pointsValuesForLetters array (loop)
+const searchMainArray = (mainArray, questionedLetter4) => {
+  const outerArray = mainArray
+  const questionedLetter3 = questionedLetter4
+  for (let i = 0; i <= outerArray.length - 1; i++) {
+    // each time - call function that searches that element (inner array)
+    searchInnerArray(outerArray[i], questionedLetter3)
```

Comment:
I'm curious to know why you have an "outer" array and an "inner" array?

Reply:
I was trying an approach where the letters and associated scores are in an array, within an array of all possible scores